### DISCUSSION TITLE
Please allow CSS and JS files to be loaded into the page.

### DISCUSSION THREAD

**Discussion by ialbert** [retypeapp/retype#205] [Ideas]
I am evaluating `retype` for a site used in education and I would suggest allowing custom CSS/JS files to be easily included as assets in the project configuration section. Something like:

```
css: [style1.css, style2.css]
js: [script1.js,script2.js]
```

From an implementation perspective, it seems like a fairly simple task, but it would go a long way in allowing people to tweak and customize their sites to their liking.

While the default styling is pleasing, and overall there is benefit in retaining a similar look when all sites look alike all of them become a little more boring. I remember when seemingly every site looked like the default Bootstrap theme, it was a bit of a letdown.

All it takes is a little customization, here and there to spice up the look and make each site have more personality. For example, one might want to use Prism.js for syntax highlighting, or use a different color scheme.

Evidently, one could already inject CSS and JS with a post-processing step on the HTML but it would be more elegant if the system supported that behavior.

**Comment by geoffreymcgill** [comment]
Hi @ialbert. Thanks for the feedback. We are definitely working on some options and those features will be included in `v1.12` and future releases. We want to make it super simple to customize the color palette but also provide more fine-grained control for those that require it.

We're still working out how custom JavaScript can be added to the page.

> For example, one might want to use Prism.js for syntax highlighting, or use a different color scheme.

Retype uses Prisim.js for syntax highlighting. See the [code block](https://retype.com/components/code-block/).

Different color schemes will be available in `v1.12`.

**Reply by DannyBen** [reply]
Hmm.... so I see there is no way to inject a custom javascript to the header?

I was looking for ways to [customize the plausible host](https://github.com/retypeapp/retype/discussions/272#discussioncomment-2878858).

**Reply by geoffreymcgill** [reply]
You can inject any custom HTML into the `<head>` by using the `_includes/head.html` include. More details available in the following thread:

https://github.com/retypeapp/retype/discussions/282#discussioncomment-2343005

Hope this helps.
